### ISSUE TITLE
Patch release broke working repo.

### ISSUE THREAD

**Issue by greg-hellings** [ansible/ansible-lint#2645]
##### Summary

Version 6.8.5 broke code which previously was passing with 6.8.4.

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

Github Actions Ubuntu runner

##### STEPS TO REPRODUCE

See [this run](https://github.com/devroles/ansible_collection_system/actions/runs/3366655214/jobs/5583360534) of dependabot upgrades that resulted in a broken run.

##### Desired Behavior

Patch-level changes should not break existing code.

##### Actual Behavior

Patch release broke existing code with confusing error message. Error message states that requirements.txt does not exist, but it does and has existed.

**Comment by sverrehu** [comment]
We see the same. Several messages like this:

```text
ERROR! Unexpected Exception, this is probably a bug: [Errno 2] No such file or directory: 'verify.yml'
```

for various files and directories that are there.

**Comment by greg-hellings** [comment]
This behavior is still present in 6.8.6. [See here](https://github.com/devroles/ansible_collection_system/actions/runs/3374627875/jobs/5600446304) where the error is still appearing. Now it is complaining about `defaults` instead of `requirements.txt`.
